src/ext-test/java/org/opentripplanner/ext/flex/FlexIntegrationTest.java

```diff
@@ -67,6 +67,11 @@ static void setup() {
     service = TestServerContext.createServerContext(graph, transitModel).routingService();
   }
 
+  @Test
+  public void addFlexTripsAndPatternsToGraph() {
```

Comment:
Not directly related to the PR, but this can be an opportunity to remove the public modifier on the test class and methods (unnecessary with JUnit 5)

Reply:
Thanks for the tip. I've removed the `public` modifier in all of the flex tests.